src/Interface/Application/SCIRunMainWindow.cc

```diff
@@ -102,33 +102,6 @@ SCIRunMainWindow::SCIRunMainWindow()
 
   setAttribute(Qt::WA_DeleteOnClose);
 
-  setStyleSheet(
-    "background-color: rgb(66,66,69);"
-    "color: white;"
-    "selection-color: yellow;"
-    "selection-background-color: blue;"//336699 lighter blue
-    "QToolBar {        background-color: rgb(66,66,69); border: 1px solid black; color: black;     }"
-    "QProgressBar {        background-color: rgb(66,66,69); border: 0px solid black; color: black  ;   }"
-    "QDockWidget {background: rgb(66,66,69); background-color: rgb(66,66,69); }"
-    "QPushButton {"
```

Comment:
@RubioJr9 These QPushButton stylesheet options seem to have been lost. Did they not apply anywhere so you removed them? Or were they mistakenly not copied to the app level?

Reply:
I'll double check but I believe they didn't have any visible effect.